lib/src/main/scala/spinal/lib/bus/regif/BusIfBase.scala

```diff
@@ -26,16 +31,106 @@ trait BusIfBase extends Area{
 
   def busDataWidth: Int
   def wordAddressInc: Int = busDataWidth / 8
+  def strbWidth: Int = busDataWidth / 8
+  def underbitWidth: Int = log2Up(wordAddressInc)
+
+  def mwdata(sec: Range): Bits = if(withstrb) writeData(sec) & wmask(sec) else writeData(sec)
```

Comment:
Ahh, it's really easy to get confused with "wdata", "mwdata" and "writeData". ^^